src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -3333,7 +3333,7 @@ void ObjectMgr::LoadPlayerInfo()
                     continue;
                 }
 
-                int32 amount   = fields[3].GetInt8();
+                int32 amount   = fields[3].GetUInt16();
```

Comment:
@Kitzunu  I didn't notice this but why are we storing an unsigned int inside a signed `int32` ?